src/main/java/io/appium/java_client/remote/AppiumW3CHttpCommandCodec.java

```diff
@@ -51,7 +42,6 @@ public AppiumW3CHttpCommandCodec() {
         defineCommand(GET_ELEMENT_ATTRIBUTE, get("/session/:sessionId/element/:id/attribute/:name"));
         defineCommand(IS_ELEMENT_DISPLAYED, get("/session/:sessionId/element/:id/displayed"));
         defineCommand(GET_PAGE_SOURCE, get("/session/:sessionId/source"));
-        defineCommand(SEND_KEYS_TO_ACTIVE_ELEMENT, post("/session/:sessionId/actions"));
```

Comment:
`org.openqa.selenium.remote.DriverCommand.SEND_KEYS_TO_ACTIVE_ELEMENT` is removed in https://github.com/SeleniumHQ/selenium/commit/4dcc9c44f1afcf6cdd1b7d76dc9115358b29c2ea